website/docs/cli/commands/auth/usage.mdx

```diff
@@ -0,0 +1,566 @@
+
+  identities:
+    saml-admin:
+      kind: aws/assume-role
+      default: true
+      via:
+        provider: okta-saml
+      principal:
+        assume_role: arn:aws:iam::123456789012:role/AdminRole
+```
+
+### GitHub Actions OIDC
+
+```yaml
+auth:
+  providers:
+    github-oidc:
+      kind: github/oidc
+      region: us-east-1 # Required
+      spec:
+        audience: sts.us-east-1.amazonaws.com
+
+  identities:
+    github-deploy:
+      kind: aws/assume-role
+      default: true
+      via:
+        provider: github-oidc
+      principal:
+        assume_role: arn:aws:iam::123456789012:role/GitHubActionsRole
+```
+
+### AWS User (Break-glass)
+
+```yaml
+auth:
+  identities:
+    emergency-user:
+      kind: aws/user
+      credentials:
+        access_key_id: !env EMERGENCY_AWS_ACCESS_KEY_ID
+        secret_access_key: !env EMERGENCY_AWS_SECRET_ACCESS_KEY
+        region: us-east-1
+```
+
+Alternatively
+
+```yaml
+auth:
+  identities:
+    emergency-user:
+      kind: aws/user
+      credentials:
+        region: us-east-1
+```
+
+Then run `atmos auth user configure` to configure the credentials on the keychain. See [user configure](/cli/commands/auth/user-configure) for details.
+
+## CLI Commands
+
+### Authentication Commands
+
+```bash
+# Validate auth configuration
+atmos auth validate
+atmos auth validate --verbose
+
+# Login (authenticate and cache credentials)
+atmos auth login
+atmos auth login --identity prod-admin
+
+# Check current authentication status
+atmos auth whoami
+atmos auth whoami --identity dev-admin
+
+# Get environment variables
+atmos auth env
+atmos auth env --identity prod-admin --format bash
+atmos auth env --format json
+atmos auth env --format dotenv
+atmos auth env --format dotenv
+
+# Execute command with authentication
+atmos auth exec --identity prod-admin -- aws sts get-caller-identity
+atmos auth exec -- terraform plan
+
+# Configure AWS user credentials
+atmos auth user configure
+```
+
+## Component-Level Configuration
+
+Override authentication settings for specific components:
+
+```yaml
+# In your component configuration
+components:
+  terraform:
+    myapp:
+      auth:
+        identities:
+          custom-role:
+            kind: aws/assume-role
+            via:
+              provider: my-sso
+            principal:
+              assume_role: arn:aws:iam::123456789012:role/MyAppRole
+```
+
+## Environment Variable Formats
+
+### Bash Format
+
+```bash
+atmos auth env --format bash
+# Output:
+export AWS_ACCESS_KEY_ID="AKIA..."
+export AWS_SECRET_ACCESS_KEY="..."
+export AWS_SESSION_TOKEN="..."
+```
+
+### JSON Format
+
+```bash
+atmos auth env --format json
+# Output:
+{
+  "AWS_ACCESS_KEY_ID": "AKIA...",
+  "AWS_SECRET_ACCESS_KEY": "...",
+  "AWS_SESSION_TOKEN": "..."
+}
+```
+
+### Dotenv Format
+
+```bash
+atmos auth env --format dotenv
+# Output:
+AWS_ACCESS_KEY_ID=AKIA...
+AWS_SECRET_ACCESS_KEY=...
+AWS_SESSION_TOKEN=...
+```
+
+## Default Identity Handling
+
```

Comment:
```suggestion

A **default identity** is the one Atmos Auth will use automatically when no specific identity is requested.  

- If you configure a single identity and mark it as `default: true`, Atmos will always use it without requiring you to pass `--identity`.  
- If multiple identities are defined, you can still mark one as default, but you’ll need to explicitly choose another when you don’t want the default.  
- If no default is set and multiple identities exist, Atmos will require you to specify which identity to use.  

This makes workflows simpler because you can avoid passing extra flags for the identity you use most often.

```